src/utils/patches.js

```diff
@@ -387,3 +392,50 @@ export const addPrefixToPatch = (prefix, patch) => ({
   ...patch,
   path: `${prefix}${patch.path}`,
 });
+
+const getWrappedDeviceType = device => {
+  let type;
+  if (isNic(device)) {
+    type = DEVICE_TYPE_INTERFACE;
+  } else if (isDisk(device)) {
```

Comment:
I know this was needed historically but `isNic` and `isDisk` are not needed anymore. You can just call `get(device, 'type')` instead of this method,

Reply:
Good point. Done!